ansible/templates/etc/systemd/system/deploy-static-sticky.service.j2

```diff
@@ -2,10 +2,10 @@
 
 [Unit]
 Description=Deploy static-sticky
-OnFailure=failure-notificator@%n.service
```

Comment:
Why remove this?

Reply:
Because that would send a duplicate failure message to #alerts, while the "failed" deployment status already exposes an event that should be posted to a channel for the CommIT.